pkg/helpers/config/userdata.yaml

```diff
@@ -1,13 +1,13 @@
 #cloud-config
-repo_update: true
-package_update: true
-packages:
-  - docker 
 runcmd:
-  - sudo service docker start
+  # taking care manually of the packages to be able to make it non-blocking
```

Comment:
Would it be ok to make updates non-blocking if we want to enforce an update e.g. if there's a security update included? Can we switch blocking failed updates using an input flag?

Reply:
We can card it if we feel it is needed, but I didn't as I don't believe there is a strong usecase : 
- the default AMI will be used by OCM without updates, so we need to be fine with using this AMI on customer account
- when using the cli, we already have the ability to specify the ami to use if we face a specific scenario (and golden-ami is installing up-to-date packages).

As a side note, I still need to create a card for the maintenance of the AMI with regard to security patchs and to get rid of the explicit AMIs in the code which is the longer term solution for the security patching. But I am not sure this should be blocking here.